assets/src/block-templates/homepage/template.js

```diff
@@ -0,0 +1,34 @@
+import gravityFormWithText from '../templates/gravity-form-with-text';
+const {__} = wp.i18n;
+const isNewIdentity = window.p4ge_vars.planet4_options.new_identity_styles ?? false;
+const backgroundColor = isNewIdentity ? 'beige-100' : 'grey-05';
+
+const template = () => (
+  [
+    ['core/group', {}, [
+      ['planet4-blocks/carousel-header'],
+      ['planet4-block-templates/issues', {
+        titlePlaceholder: __('The issues we work on', 'planet4-blocks-backend'),
```

Comment:
I see some inconsistencies in the translation domains, I think `planet4-blocks` makes more sense since these might be visible in the frontend too if editors don't change them? 🤔

Reply:
Although some are obvious placeholders like `Enter title` so I'm not sure

Reply:
All the strings that appear in the frontend should be just `planet4-blocks`. But I think input placeholders don't.

Reply:
Yea I think so too, because in the [php](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/classes/patterns/class-homepage.php#L45) file it was `planet4-blocks-backend`

Reply:
Yeah, this case is a bit tricky because technically it's not really a placeholder (not as generic as `Enter title` for example), editors might not change it and in that case it will appear in the frontend, that's why I wasn't sure